roles/ceph-mon/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# You can override vars by using host or group vars
+
+###########
+# GENERAL #
+###########
+
+# ACTIVATE BOTH FSID AND MONITOR_SECRET VARIABLES FOR NON-VAGRANT DEPLOYMENT
+fsid: "{{ cluster_uuid.stdout }}"

```

Comment:
Within Ursula we use the hash merge option, so that group_vars /can/ override namespaced variables.